open_prices/api/prices/filters.py

```diff
@@ -31,6 +31,9 @@ class PriceFilter(django_filters.FilterSet):
         field_name="proof__type",
         choices=proof_constants.TYPE_CHOICES,
     )
+    proof__owner_consumption = django_filters.BooleanFilter(
```

Comment:
actually I don't know if it's the right strategy...
because it'll be tricky to filter for the opposite ("community" prices).
maybe instead create a new custom filter - for instance `proof__group` that could be "consumption" or "community", and we would manage the custom filtering somewhere in the backend